server/recceiver/recast.py

```diff
@@ -241,6 +241,27 @@ def __str__(self):
             source_address, init, conn, nenv, nadd, nalias, ninfo, ndel
         )
 
+    def __repr__(self):
+        return """Transaction(
+            source_address={},
+            initial={},
+            connected={},
+            records_to_add={},
+            client_infos={},
+            record_infos_to_add={},
+            aliases={},
+            records_to_delete={})
+            """.format(
+            self.source_address,
+            self.initial,
+            self.connected,
+            self.records_to_add,
+            self.client_infos,
+            self.record_infos_to_add,
+            self.aliases,
+            self.records_to_delete,
```

Comment:
Maybe change this to use an f-string (for readability), and then use `self.__class__.__name__` for the class name (for maintainability)? It would be nice if we also could do `{var=}`, but unfortunately that seems to print also `self` in this case.